New Personal Website/contact.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <h3>Contact details</h3>
+    <ul>
+      <li><em><strong>Phone Number</strong></em>:7022814594</li>
+      <li><em><strong>Email</strong></em>:rahul4132@gmail.com</li>
+    </ul>
+    <form class="Query Form" action="mailto:rahul4132@gmail.com" method="post" enctype="text/plain">
```

Comment:
oh cool does this actually work?

Reply:
so what happens here is that it opens a mailing software like outlook or your browser with a drafted mail. All you have to do is send.